content/blog/scverse_announcement.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "scverse Announcement"
+date = 2022-01-19T14:28:05+01:00
+description = "Introducing scverse"
+draft = false
++++
+
+With the increasing reliance of scientific research on infrastructural and analytical software, foundational tools in life sciences have outgrown individual labs and institutes in terms of their scale and impact. That has encouraged us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem.
```

Comment:
I would not go as broad as life sciences. I would stick to single cell, this is our audience anyway

Reply:
This is actually the phrasing we devised together with the management committee. We see this targeting more than just single cells already now and in future, also AnnData / Scanpy / etc. can be used for bulk analysis just as well.

Reply:
I also think it would be fine to limit the scope to single-cell for now? It doesn't prevent us from extending to other domains later anyway, should that ever become relevant.

Reply:
I mean it's literally in our name: **sc**verse

Reply:
I'd say the first sentence should focus on single cell since it is our current focus. After that, I think it's fine to be a bit loose with scope otherwise.